numpy/_core/tests/test_multiarray.py

```diff
@@ -4025,6 +4025,18 @@ class LowPriority(np.ndarray):
         assert res.shape == (3,)
         assert res[0] == 'result'
 
+    @pytest.mark.parametrize("scalar", [
+            np.longdouble(1), np.timedelta64(120, 'm')])
+    @pytest.mark.parametrize("op", [operator.add, operator.xor])
+    def test_scalar_binop_guarantees_ufunc(self, scalar, op):
+        # Test that __array_ufunc__ will always cause ufunc use even when
+        # we have to protect some other calls from recursing (see gh-26904).
+        class SomeClass:
+            def __array_ufunc__(self, ufunc, method, *inputs, **kw):
+                return "result"
+
+        assert SomeClass() + np.longdouble(1) == "result"
+        assert np.longdouble(1) + SomeClass() == "result"
```

Comment:
```suggestion
        assert SomeClass() + scalar == "result"
        assert scalar + SomeClass() == "result"
```

?

Reply:
yes, had started with that and then apparently only added the parametrize half.  Thanks.